### DISCUSSION TITLE
关于app意见及建议

### DISCUSSION THREAD

**Discussion by xiaozhu0126** [gyc-12/Cymusic#75] [General]
软件非常吊，白嫖党至上 love

希望能出一个历史播放歌曲的类目，首页三横的功能希望可以移至右下角，变成一个全功能型的菜单， 其他权限希望尽快开放，安卓和pc端，iospc端也快块来吧  支持！！！

**Comment by mnn666** [comment]
建议播放英文歌曲时歌词可以显示中文翻译，祝愿越来越好

**Reply by ejungd** [reply]
1.17版，搜索框还是会盖住歌单

**Comment by mnn666** [comment]
作者大哥，有个小小的建议，希望优化一下，就是喜欢和下载里的歌曲顺序应该是逆向的，按照54321这样的往上排列的，现在115版本的是正向的，是12345这样的，比如，我现在下载了一首歌1，过两天又下载了一首歌2，2的位置应该在1的上面，而不是下面的，不知道我描述清楚了没？

**Comment by mnn666** [comment]
遇到无高音质的音乐自动尝试切换低一级音质播放，如果再听其他高音质的会变成高音质吗？

**Comment by liruirui1218** [comment]
尊敬的开发者你好，我很喜欢这个cymusic应用程序，我在使用中发现一个问题，在cymusic播放音乐时，切换到微信听完微信语音，音乐就自动暂停了，不能继续播放音乐了。我想让他在听完语音后也像其他音乐应用程序一样继续播放音乐。
  谢谢！

**Comment by Kokouki** [comment]
开发一个 Mac 版的 app 吧，谢谢

**Comment by zj30635315** [comment]
源太少了，各位有好源分享一下吗？谢谢
